libdnf/rpm/repo_impl.hpp

```diff
@@ -92,8 +80,62 @@ class Key {
     long int timestamp;
 };
 
+// Information about attached libsolv repository
+class LibsolvRepoExt {
+public:
+    enum class DataType { FILENAMES, PRESTO, UPDATEINFO, OTHER };
+    enum class DataState { NEW, LOADED_FETCH, LOADED_CACHE, WRITTEN };
+
+    Id get_data_id(DataType which) const noexcept;
```

Comment:
Please could you add a comment what is returning. Id or int is not informative enough.